Automattic/wp-calypso
# Change wordings and a bit of style for p2 creation

#### Changes proposed in this Pull Request
This changes some slightly confusing wording and a tiny bit of styling on the p2 creation form. 

See 4529-gh-Automattic/p2 and 4531-gh-Automattic/p2

#### Testing instructions
Go to `http://calypso.localhost:3000/start/p2/p2-site` and look at the changes there. You'll see changes to the "Hooray" screen once you submit the form.



Before:
<img width="474" alt="Screen Shot 2022-01-28 at 16 49 27" src="https://user-images.githubusercontent.com/193283/151632394-eed2e0c2-c9b9-4061-9097-c02daef6f83b.png">

After:

<img width="525" alt="Screen Shot 2022-01-28 at 16 43 20" src="https://user-images.githubusercontent.com/193283/151632398-447e4940-bfaf-4a38-b50b-24ce404d0e17.png">



Related to #
